# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini
# Commit: e9ee515307b2
# Full Hash: e9ee515307b2acef4ce4ad88ca2e9d8a9aa781a8
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2023-04-23 09:12:41
# Regressor Bug: 1829397
# File Overlap Count: 3
# Description:
#   Bug 1829397 [wpt PR 39645] - Update wpt metadata, a=testonly
#   
#   wpt-pr: 39645
#   wpt-type: metadata
# ==============================================================================

diff -r 73832b4dfc98 -r e9ee515307b2 testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/fenced-frame/sandbox-mandatory-flags.https.html.ini	Fri Apr 21 19:57:56 2023 +0000
@@ -0,0 +1,12 @@
+[sandbox-mandatory-flags.https.html]
+  [Sandboxed Iframe with mandatory flags can load a fenced frame.]
+    expected: FAIL
+
+  [Sandboxed Iframe without one of mandatory flag must fail to load a fenced frame.]
+    expected: FAIL
+
+  [Nested sandboxed iframe without one of mandatory flag must fail to load afenced frame even when the inner nested sandboxed iframe has all mandatory allow- flags.]
+    expected: FAIL
+
+  [navigator.canLoadAdAuctionFencedFrame considers mandatory sandbox flags]
+    expected: FAIL